### ISSUE TITLE
2023-09-15/14: Smart Maps Meetup Pacific 6

### ISSUE THREAD

**Issue by hfu** [UNopenGIS/7#247]
We revisit this issue and add information after #246. 
The last session of the Smart Maps Meetup Monthly (or Smart Maps Meetup Pacific) was #219.

**Comment by hfu** [comment]
@albertkun Do you think we can invite a guest speaker for this meetup from California? Otherwise, we may postpone the Pacific meetup, or we can use this opportunity for something like AAR (After-Action Review) like sharing what was successful, and what needs to be improved. What do you think?

**Comment by albertkun** [comment]
@hfu both are good ideas! I can confirm the guest speaker sometime next week if that is OK with you! Otherwise, I agree that we should use the time for an AAR! Thank you for following up!

**Comment by hfu** [comment]
@albertkun Thank you! Please take your time!

**Comment by albertkun** [comment]
@hfu You are welcome! @matikin9 has tentatively confirmed a speaker! I will get the details later this week when she comes back from a trip!

**Comment by albertkun** [comment]
@hfu can we move this meeting to 12pm JST once again? The speakers will be from the east coast, so aj earlier time would be best!

**Comment by hfu** [comment]
@albertkun Let's arrange to do so. It would definitely be necessary to do so if we collaborate with the east coast.

**Comment by albertkun** [comment]
@hfu excellent! I will update the Google Calendar/Zoom meeting later this week!

**Comment by albertkun** [comment]
@hfu received the details of the guest speaker and will be updating the flier by Monday!  Also, since I am still in Japan until 2023-09-11 I think it would be nice to meet up with @hfu @yuiseki @smellman again! Maybe in Tsukuba? No problem if people are too busy though!

**Comment by albertkun** [comment]
@hfu here is the updated flier! I will be working on the portrait mode later today!
![unsmartmaps-flyer-2023-09-14](https://github.com/UNopenGIS/7/assets/8574425/2d189ceb-b7e5-4ad3-96d0-175f167d3891)

**Comment by hfu** [comment]
Thanks @albertkun the QR code is also cool! I will advertise this with my friends! 

By the way, we discussed about our face-to-face meeting in Japan on #265. Unfortunately there may not be a good time for all of us to meet together. Maybe we can concentrate on the Pacific meetup and Documentation meeting.

**Comment by albertkun** [comment]
@hfu Great! I will go ahead and do the same as well! Let's try to standardize the hashtags so we can build up some momentum! 

No problem about not being able to meet up, hopefully we can see each other again next year! :) 

Also, maybe we can rebrand the "Documentation" meeting into a core-group meeting where we can discuss about the logistics/direction of the group: we can use the the time to talk about items such as the "core mission statement", "marketing", etc. and we can coordinate the short term and long term goals/events with those that are interested/have the capacity to contribute. Let me know what you think!

**Comment by hfu** [comment]
@albertkun I agree on the rebranding of "Documentation" meeting into direction meeting because we tend to discuss direction issues when we discuss about documentation. I feed it is a very strategic to think about documentation, direction, and marketing in an indivisible way.

**Comment by albertkun** [comment]
@hfu Really sorry, would it be OK for you to step in to run this month's meeting again? 

Last week I planned to return to USA, but I got really sick (the flier was the last thing I did before I got hit with a huge headache). And just as I was about to head out I tested positive for COVID-19, so I don't think I have much time to bandwidth. Please let me know what you think.

**Comment by hfu** [comment]
@albertkun Many thanks for your great contribution! I am happy to be prepared for being the substitute in case. 

I also think that the idea of **rebranding of "Documentation" meeting into direction meeting** is huge. I am sharing this in the upcoming weekly meeting and will create a separate issue on this. 

Take care. I wish you will get well soon!

**Comment by albertkun** [comment]
@hfu Thank you so much!! I am very appreciative as its been tough with covid. Here is the link the the markdown that you can edit if you wish to do so:
https://hackmd.io/@c55bL-YKSuS4Vu7yIWZSdg/B1harN3Cn/edit

In terms of the "direction" meeting, I referred to it as the "Core Committee," meeting but we can discuss what to call it in #270!

Thank you and I will spend my time trying to rest as much as possible!

**Comment by hfu** [comment]
@albertkun Thank you again for your great contribution to this meetup. Without your contribution it was not possible. I appreciate it.

**Comment by albertkun** [comment]
@hfu you are very welcome! I also very much appreciate you stepping in and running the meeting while I was still sick 😷 Thank you so much for that!

@smellman here is the recording, if you can upload it that would be great!

https://ucla.zoom.us/rec/share/r1ldUvdKScjyvFuMX_6c32gxXS4NwSb3mU2kjbRdCk2bfvp9rT7xyffOqOX-Qus2.nisQ2-yTzD9TL8OC
